frequency-plans.yml

```diff
@@ -159,6 +159,14 @@
   country-codes: [ar, au, bo, br, ca, cl, co, do, nz, py, pe, sr, uy]
   file: AU_915_928_FSB_8.yml
 
+- id: AU_915_928_FSB_1_NDT
+  band-id: AU_915_928
+  name: Australia 915-928 MHz, FSB 1
+  description: Default frequency plan for Australia, using sub-band 1 and dwell time disabled
```

Comment:
Thanks for clarifying @EduardoPfeifer; I already assumed it would be for use outside Australia. This description is therefore not valid.